inclusion_connect/settings/base.py

```diff
@@ -298,9 +298,12 @@
 # Django-oauth-toolkit
 # --------------------
 
+with open("oidc.pem", "r") as rsa_key_file:
+    OIDC_RSA_PRIVATE_KEY = rsa_key_file.read()
```

Comment:
Les minuscules permettent de ne pas exposer la clé dans les settings de django directement.
https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text
```suggestion
from pathlib import Path
oidc_rsa_private_key = Path("oidc.pem").read_text()
```